### PR TITLE
fix Jenkins pulsar-website-build error 571

### PR DESCRIPTION
**Pull request by ambition119**
### Motivation
Jenkins pulsar-website-build error 571， error info:
```java
Error: No 'original_id' field found in /pulsar/site2/website/versioned_docs/version-2.3.0/io-cdc-canal.md. Perhaps you forgot to add it when importing prior versions of your docs?
    at /pulsar/site2/website/node_modules/docusaurus/lib/server/versionFallback.js:64:11
    at Array.forEach (<anonymous>)
    at Object.forEach (/pulsar/site2/website/node_modules/docusaurus/lib/server/versionFallback.js:52:7)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Module._compile (/pulsar/site2/website/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Object.newLoader [as .js] (/pulsar/site2/website/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
error Command failed with exit code 1.
```

### Modifications

io-cdc doc add original_id content.

### CODE REVIEW

**Comment by ambition119** [comment]
run Integration Tests

File: site2/website/versioned_docs/version-2.3.0/io-cdc-canal.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: io-cdc-canal
```
Comment (sijie) [inline site2/website/versioned_docs/version-2.3.0/io-cdc-canal.md]: this should be `version-2.3.0-io-cdc-canal`

File: site2/website/versioned_docs/version-2.3.0/io-cdc-debezium.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 id: io-cdc-debezium
```
Comment (sijie) [inline site2/website/versioned_docs/version-2.3.0/io-cdc-debezium.md]: this should be `version-2.3.0-io-cdc-debezium`

File: site2/website/versioned_docs/version-2.3.0/io-cdc.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 id: io-cdc
```
Comment (sijie) [inline site2/website/versioned_docs/version-2.3.0/io-cdc.md]: this should be `version-2.3.0-io-cdc`

File: site2/website/versioned_docs/version-2.3.0/io-cdc-canal.md (hunk 1 above)
Comment (ambition119) [reply site2/website/versioned_docs/version-2.3.0/io-cdc-canal.md]: > this should be `version-2.3.0-io-cdc-canal`

ok

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by ambition119** [comment]
run java8 tests

**Comment by ambition119** [comment]
run java8 tests

**Comment by ambition119** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**